draft-ietf-httpbis-digest-headers.md

```diff
@@ -148,18 +152,38 @@ This document describes Digest integrity for HTTP and is structured as follows:
 ## Concept Overview
 
 This document defines the `Digest` request and response header and trailer
-field. At a high level the value contains a checksum, computed over
-`selected representation data`
-(Section 3.2; {{!SEMANTICS=I-D.ietf-httpbis-semantics}}),
-that the recipient can use to validate integrity. `Digest` supports
-algorithm agility. The `Want-Digest` field allows endpoints to express interest
-in `Digest` and preference of algorithms.
-
-Basing `Digest` on the selected representation makes it straightforward to apply
-it to use-cases where the transferred data requires some sort of
-manipulation to be considered a representation, or conveys a partial
-representation of a resource, for example Range Requests (see Section 14.2 of
-{{SEMANTICS}}).
+field, see {{digest}}. At a high level the value contains a checksum, computed
+over `selected representation data` (Section 3.2;
+{{!SEMANTICS=I-D.ietf-httpbis-semantics}}), that the recipient can use to
+validate integrity. Basing `Digest` on the selected representation makes it
+straightforward to apply it to use-cases where the transferred data requires
+some sort of manipulation to be considered a representation, or conveys a
+partial representation of a resource, for example Range Requests (see Section
+14.2 of {{SEMANTICS}}).
```

Comment:
```suggestion
some sort of manipulation to be considered a representation or conveys a
partial representation of a resource, such as Range Requests (see Section
14.2 of {{SEMANTICS}}).
```
There are only two items here, so you don't need a comma before "or"; "for example" would need a comma after it as well, and that would be a lot of commas.